product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.2-Modify-payload-by-adding-a-sibling-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddSiblingTest.java

```diff
@@ -34,14 +34,14 @@
  */
 public class AddSiblingTest extends ScenarioTestBase {
```

Comment:
Shall we rename the Testcase as well the test methods to be more meaningful? e.g., AddSiblingUsingEnrichMediatorTest? This might be usefule for tracking purposes in the future.